senders/pushover/pushover.go

```diff
@@ -71,21 +75,34 @@ func (sender *Sender) makePushoverMessage(events moira.NotificationEvents, conta
 	return pushoverMessage
 }
 
-func (sender *Sender) buildMessage(events moira.NotificationEvents, throttled bool) string {
+func (sender *Sender) buildMessage(events moira.NotificationEvents, throttled bool, trigger moira.TriggerData) string {
 	var message bytes.Buffer
-	for i, event := range events {
-		if i > printEventsCount-1 {
-			break
-		}
-		message.WriteString(fmt.Sprintf("%s: %s = %s (%s to %s)", event.FormatTimestamp(sender.location), event.Metric, event.GetMetricValue(), event.OldState, event.State))
+	desc := trigger.Desc
+	if len(desc) > descLimit {
+		// trim description to fit 512 chars while leaving space for html tags
```

Comment:
It's a bad idea to write 512 in this comment. Constant values can change, and it will lead to confusion if somebody forgets to edit this comment in the future.